### PR TITLE
Update README to reflect changes made during application development

### PR DESCRIPTION
**Pull request by amoriartyCH**
Add mention of new docs directory
Update the environment variables table to include the log level
Remove reference to swagger specs as that is for swagger & Java

**Resolves DSND-142**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,33 +4,35 @@ Service to send delta events from CHIPS on the correct kafka topic
...
+| BIND_ADDR                         | 5010                              | Bind Address / application port                    | TRUE                                            |
+| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      | TRUE                                            |
+| SCHEMA_REGISTRY_URL               | http://chs-kafka:8081             | Schema registry URL                                | TRUE                                            |
+| OFFICER_DELTA_TOPIC               | officers-delta                    | Officer Delta Kafka topic to write messages to     | TRUE                                            |
+| OPEN_API_SPEC                     | ./apispec/apispec.yml             | OpenAPI schema location                            | TRUE                                            |
+| LOG_LEVEL                         | trace                             | The level at which the logger prints               | FALSE                                           |
```
Comment (ch-tstrickland) [inline README.md:14]: Do we know what value this defaults to if it is not set?

File: README.md [hunk 2]
```diff
@@ -4,33 +4,35 @@ Service to send delta events from CHIPS on the correct kafka topic
...
 
 Development mode is available for this service in Docker CHS Development.
 
 `./bin/chs-dev development enable chs-delta-api`
 
-Swagger documentation is available for this service in the docker CHS development
-
-Navigate to http://api.chs.local/api-docs/chs-delta-api/swagger-ui.html
-
 ## Running locally without Docker CHS
```
Comment (ch-tstrickland) [inline README.md:27]: Perhaps "Running locally in Docker, without Docker CHS

File: README.md [hunk 3]
```diff
@@ -4,33 +4,35 @@ Service to send delta events from CHIPS on the correct kafka topic
...
+| BIND_ADDR                         | 5010                              | Bind Address / application port                    | TRUE                                            |
+| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      | TRUE                                            |
+| SCHEMA_REGISTRY_URL               | http://chs-kafka:8081             | Schema registry URL                                | TRUE                                            |
+| OFFICER_DELTA_TOPIC               | officers-delta                    | Officer Delta Kafka topic to write messages to     | TRUE                                            |
+| OPEN_API_SPEC                     | ./apispec/apispec.yml             | OpenAPI schema location                            | TRUE                                            |
+| LOG_LEVEL                         | trace                             | The level at which the logger prints               | FALSE                                           |
 
 ## Running Locally with Docker CHS
 Clone Docker CHS Development and follow the steps in the README.
```
Comment (ch-tstrickland) [inline README.md:17]: Needs a link to Docker CHS Development.

File: README.md [hunk 4]
```diff
@@ -4,33 +4,35 @@ Service to send delta events from CHIPS on the correct kafka topic
...
 
-Swagger documentation is available for this service in the docker CHS development
-
-Navigate to http://api.chs.local/api-docs/chs-delta-api/swagger-ui.html
-
 ## Running locally without Docker CHS
-Pull image from private CH registry by running docker pull 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/chs-delta-api:latest command or run the following steps to build image locally:
+Export the required environment variables to ensure the service can start up correctly.
+
+Pull image from private CH registry by running docker pull 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/chs-delta-api:latest
```
Comment (ch-tstrickland) [inline README.md:30]: Perhaps put the "docker pull" in command format?

**Review by ch-tstrickland — COMMENTED** [review]
This all looks good. I think that a small section demonstrating how to run Postman or similar to test that your app is up-and-running would also be useful, either here or linked to from here.

File: README.md [hunk 5]
```diff
@@ -4,33 +4,40 @@ Service to send delta events from CHIPS on the correct kafka topic
...
-| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      |
-| SCHEMA_REGISTRY_URL               | http://chs-kafka:8081             | Schema registry URL                                |
-| OFFICER_DELTA_TOPIC               | officers-delta                    | Kafka topic to write messages to                   |
-| OPEN_API_SPEC                     | ./apispec/apispec.yml             | OpenAPI schema location                            |
-
+|  Variable                         |  Example                          |  Description                                       |  Required        | Default value |        
+| --------------------------------- | --------------------------------- | -------------------------------------------------- | ---------------- | ------------- |
+| BIND_ADDR                         | 5010                              | Bind Address / application port                    | TRUE             |               |
+| KAFKA_BROKER_ADDR                 | chs-kafka:9092                    | Kafka broker address (can be comma separated)      | TRUE             |               |
```
Comment (faisal-amin82) [inline README.md:10]: I feel it will be more clearer to wider audience if we change the Required values to `YES` or `NO` compared to `TRUE` or `FALSE`

**Review by faisal-amin82 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (amoriartyCH) [reply README.md]: Added, good suggestion thank you!

**Pull request merged**